assets/js/implicit-login.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener( 'DOMContentLoaded', function() {
+    if ( ! window.location.hash || window.location.hash.indexOf('id_token') < 0 ) {
+        return;
+    }
+
+    var hash = window.location.hash;
```

Comment:
I don't know how much resources you are saving, but this line could be the first one and `hash` used on the `if`